03_letters/CameronSchafer/letters.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+// string to rune with counts
+func letters(s string) map[rune]int {
+	tMap := make(map[rune]int)
+
+	for i := range s {
+		tMap[rune(s[i])]++
+	}
+
+	return tMap
+}
+
+func sortLetters(m map[rune]int) []string {
+	tSlice := []int{}
```

Comment:
Would keys be a better variable name for tSlice?

i in the for-loop could be k, making it clearer we are looking over keys, and not array/slice indices.

Reply:
I updated the name for this variable. It definitely looks neater with a shorter name.